include/swift/AST/DiagnosticsSema.def

```diff
@@ -4534,7 +4534,7 @@ WARNING(property_wrapper_wrapperValue,none,
         "'projectedValue'; use of 'wrapperValue' is deprecated", ())
 WARNING(property_wrapper_init_initialValue,none,
         "property wrapper's `init(initialValue:)` should be renamed "
-        "to 'init(wrappedValue:)'; use of 'init(initialValue:)' is deprecated",
+        "to `init(wrappedValue:)`; use of `init(initialValue:)` is deprecated",
```

Comment:
This is a great example of the inconsistency: the same method was quoted with both backticks and single quotes within the same diagnostic.